### PR TITLE
Option to store relative pose between two cameras in database

### PR DESCRIPTION
**Pull request by yanxke**
See https://github.com/colmap/colmap/issues/1773.  So that the values can be written into the DB after feature matching.

### CODE REVIEW

**Comment by ahojnnes** [comment]
Thanks for your contribution. I have the same concerns as with the referenced PR #1565. Could we maybe add this feature behind an flag "compute_relative_pose" (which defaults to false) in the feature matching options?

**Comment by yanxke** [comment]
Yes, please see the latest commit.  It seems like the SIFT matching options is overloaded to contain other options and I couldn't find a better place to put this.

**Comment by ahojnnes** [comment]
Yes, the options are not clearly delineated. This looks good to me. Thank you!

File: src/estimators/two_view_geometry.cc [hunk 1]
```diff
@@ -361,6 +361,10 @@ void TwoViewGeometry::EstimateCalibrated(
                         num_inliers, *best_inlier_mask, options)) {
       config = ConfigurationType::WATERMARK;
     }
+
+    if (options.compute_relative_pose) {
+      EstimateRelativePose(camera1, points1, camera2, points2);
```
Comment (ahojnnes) [inline src/estimators/two_view_geometry.cc:366]: it seems like we could also add this to EstimateUncalibrated/EstimateHomography?

**Review by ahojnnes — CHANGES_REQUESTED** [review]

File: src/estimators/two_view_geometry.cc (hunk 1 above)
Comment (yanxke) [reply src/estimators/two_view_geometry.cc:366]: Sure, added.

**Review by ahojnnes — APPROVED** [review]

**Pull request merged**